README.md

```diff
@@ -38,17 +37,12 @@ yo playbook
 
 *Should you run into an error such as `command yo not found` it may be related to a path issue when installing Node.js via Homebrew. Please refer to the top answer on this [StackOverflow question](http://stackoverflow.com/questions/15846076/command-not-found-after-installation).*
 
-### Grunt Tasks
-##### grunt serve
-Serve your source locally into your browser. LiveReload will automatically load any changes to HTML, CSS and JavaScript that you make.
-
-##### grunt check
-Check the quality of your source with tools like [JSLint](http://www.jslint.com/), [CSSLint](http://csslint.net/) and [csscss](http://zmoazeni.github.io/csscss/).
+### Gulp Tasks
+##### gulp serve
+Serve your source locally into your browser. BrowserSync will automatically load any changes to HTML, CSS and JavaScript that you make.
 
-##### grunt build
+##### gulp build
 Build the concatenated, minified production version of the source into the `dist` directory.
 
-##### grunt deploy
-Deploy the production version of the source to [GitHub Pages](http://pages.github.com/). This ability is configurable during Playbook generation.
-
-If you choose to utilize `grunt-build-control` to deploy a GitHub Pages user/organization site, your source must live in a branch other than `master`. GitHub Pages [user/organization sites](https://help.github.com/articles/user-organization-and-project-pages#project-pages) serve the files found in the `master` branch to the browser.
```

Comment:
Is there a reason behind removing this paragraph? I like the cleaner look of the shortened description, but the tradeoff is clarity for users. I would imagine one argument for removal is that this relates more to GH-pages functionality instead of generator-playbook's.

I'm on the fence about removing this, however, as the deploy task seems geared towards GH-pages. Thoughts?